event/src/unvalidated/signed.rs

```diff
@@ -116,6 +116,7 @@ impl<D: serde::Serialize> Event<D> {
 
 /// A signed event envelope.
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)] // TODO Should this be here?
```

Comment:
since these structures come more from the JWS spec than from anything we control, it seems like maybe we'd be better off being loose here in what we accept